packages/functional/README.md

```diff
@@ -0,0 +1,28 @@
+[![npm][npm-image]][npm-url]
+[![npm-downloads][npm-downloads-image]][npm-url]
+[![semantic-release][semantic-release-image]][semantic-release-url]
+<br />
+[![code-style-prettier][code-style-prettier-image]][code-style-prettier-url]
+
+[code-style-prettier-image]: https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square
+[code-style-prettier-url]: https://github.com/prettier/prettier
+[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/functional/experimental.svg?style=flat
+[npm-image]: https://img.shields.io/npm/v/@solana/functional/experimental.svg?style=flat
+[npm-url]: https://www.npmjs.com/package/@solana/functional/v/experimental
+[semantic-release-image]: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+[semantic-release-url]: https://github.com/semantic-release/semantic-release
+
+# @solana/functional
+
+This package contains generalized functional helpers and functional helpers
+specific to Solana application components. It can be used standalone, but it
+is also exported as part of the Solana JavaScript SDK
+[`@solana/web3.js@experimental`](https://github.com/solana-labs/solana-web3.js/tree/master/packages/library).
+
+## Types
+
+TODO
+
+## Functions
+
+TODO
```

Comment:
This reminds me; I have some work to do to update some READMEs that I forgot about.